transports/p2p/peer/peer.go

```diff
@@ -1809,6 +1808,32 @@ func (p *Peer) Disconnect() {
 	close(p.quit)
 }
 
+func (p *Peer) readVerOrVerAck() error {
```

Comment:
I guess, the main purpose of this function is to check for potential error - so consider function name like: "checkVerOrVerAck". 

The same suggestion, concerning `readVerAckMsg` function

Reply:
Main purpose is to read Version or VerAck message from p2p "wire" (I don't know how to name it).
Because after sending Version from our side, we can receive either Version of VerAck message, and it seems that for us it is the best to accept both situations because we then can connect more peers.

Reply:
Ok, i get it 👍